version-compatibility/Cargo.toml

```diff
@@ -7,3 +7,7 @@ exclude = [
   "0.14.1-0.15.0/fuel-core-0.14.1-client-0.15.0",
   "0.14.1-0.15.0/fuel-core-0.15.0-client-0.14.1",
 ]
+
+# Remove when we upgrade rust version 
+[patch.crates-io]
+netlink-proto = { git = "https://github.com/rust-netlink/netlink-proto", tag = "v0.11.3" }
```

Comment:
will we be able to release to crates.io with a patch like this? curious

Reply:
No, we can't. Can we create an issue in the repository related to the crate? And on our side for now just to use old version explicitly?

Reply:
I changed on our side. Why do you want to create an issue on their repo to tell them that they increased MRSV in the last patch release ?

Reply:
If it was done in the minor release, then they need to revert it and create a new major release